library/common/jni/jni_interface.cc

```diff
@@ -30,6 +30,20 @@ extern "C" JNIEXPORT jlong JNICALL Java_io_envoyproxy_envoymobile_engine_JniLibr
   return init_engine();
 }
 
+static void jvm_on_engine_running(void* context) {
+  __android_log_write(ANDROID_LOG_VERBOSE, "[Envoy]", "jvm_on_engine_running");
+
+  JNIEnv* env = get_env();
+  jobject j_context = static_cast<jobject>(context);
+  jclass jcls_JvmonEngineRunningContext = env->GetObjectClass(j_context);
+  jmethodID jmid_onEngineRunning = env->GetMethodID(
+      jcls_JvmonEngineRunningContext, "invokeOnEngineRunning", "()Ljava/lang/Object;");
+  env->CallObjectMethod(j_context, jmid_onEngineRunning);
+
+  env->DeleteLocalRef(jcls_JvmonEngineRunningContext);
+  env->DeleteGlobalRef(j_context);
```

Comment:
Can you put a comment here? This works only because the other current engine callback doesn't use the context, but if others get added and/or when we move to per-instance runs, this will need to be moved.

Reply:
Sure, will add in a separate PR since this one is green

Reply:
https://github.com/lyft/envoy-mobile/pull/1101